includes/views/amazon-product-advertising-view.php

```diff
@@ -1,10 +1,14 @@
+<?php
+$plan    = apply_filters( 'product_feedzy_license_plan', 0 );
```

Comment:
I see that `$plan` is not used. Did you forget to remove it after you started using the filter `apply_filters( 'feedzy_is_license_of_type', false, 'business' )`?